[Chai-matchers] Add polling mechanism when automine is not available (#7997)
NomicFoundation/hardhat: .changeset/brave-tigers-wave.md

---
"@nomicfoundation/hardhat-ethers": patch
---

> Implemented `HardhatEthersProvider.waitForTransaction` with polling support for `non-automining` networks ([#7952](https://github.com/NomicFoundation/hardhat/issues/7952)).

Reviewer: ```suggestion
Added `HardhatEthersProvider.waitForTransaction` to proved polling support for `non-automining` networks ([#7952](https://github.com/NomicFoundation/hardhat/issues/7952)).
```
